src/components/00-design-tokens/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function px-to-rem($px-number) {
```

Comment:
Honestly we should replace `font-size-to-rem` and `line-height-to-rem` with this going forward.

Reply:
Yeah will do that in a future branch 👍